cpp/src/gandiva/jni/jni_common.cc

```diff
@@ -755,13 +816,15 @@ Java_org_apache_arrow_gandiva_evaluator_JniWrapper_evaluateProjector(
       uint8_t* value_buf = reinterpret_cast<uint8_t*>(out_bufs[buf_idx++]);
       jlong data_sz = out_sizes[sz_idx++];
       if (arrow::is_binary_like(field->type()->id())) {
-        buffers.push_back(std::make_shared<JavaResizableBuffer>(value_buf, data_sz));
+        buffers.push_back(std::make_shared<JavaResizableBuffer>(
```

Comment:
should we add a null check for expander in case of varlength output..

Reply:
done